lonabot/bot.py

```diff
@@ -234,6 +238,10 @@ async def start(self, update):
 
 `/tz` without an argument will show your currently configured offset.
 
+You can configure what "later" means to you with:
+`/later 1y2mo3w4d5h6s`
+Where all parts are optional but at least one has to be present.
```

Comment:
So… they're not all optional? This could use some rewording.

Reply:
I guess we can drop the optional part and just say at least one has to be present?

Reply:
Simplier examples are now used and the "all parts optional" part has been dropepd.